feat: adds Terraform Controller blog post

gberenice opened the pull request:
## Info
* Adds new content - Terraform Controller blog post, and static images used there.

Gowiem commented on content/updates/terraform-controller-overview.md:
```diff
@@ -0,0 +1,210 @@
+
+### Open Source UI
+
+Another area of interest is the Open Source UI, as outlined in the [Weave GitOps documentation](https://docs.gitops.weave.works/docs/open-source/getting-started/ui-OSS/). The ability to easily oversee and control all the details and stages of infrastructure deployment and management is always a valuable addition.
```
```suggestion
Another area of interest that we'd like to see built out further is the Open Source UI, as outlined in the [Weave GitOps documentation](https://docs.gitops.weave.works/docs/open-source/getting-started/ui-OSS/). The ability to easily oversee and control all the details and stages of infrastructure deployment and management is always a valuable addition.
```

Gowiem commented on content/updates/terraform-controller-overview.md:
```diff
@@ -0,0 +1,210 @@
+
+Despite these challenges, we still hope for the best outcome. The project is still alive, with some team members continuing their work and planning future developments. It's worth noting that the project could either be renamed and retain all repositories under the current organization or move to a different organization.
+
+We're happy to hear that the community is committed to continuing discussions about the project's future and development, and we're ready to support them in all possible ways.
```
```suggestion
We're happy to hear that the community is committed to continuing discussions about the project's future and development, and we're ready to support them as believe this tool has some real potential.
```

Gowiem commented on content/updates/terraform-controller-overview.md:
```diff
@@ -0,0 +1,210 @@
+
+## Conclusion
+
+The Terraform Controller could be a game-changer and make managing infrastructure as code in a GitOps framework a breeze. Still, we acknowledge that building a solid and usable solution requires more investments. We welcome any potential customers who are interested in learning more about this technology to reach out and chat with us!
```
```suggestion
The Terraform Controller could be a game-changer in managing infrastructure as code via GitOps a breeze. Still, we acknowledge that building a solid and usable solution requires further investment. If you're interested in this approach, we welcome any engineering orgs who are interested in using this tool to [reach out and chat with us](https://masterpoint.io/contact/) as we'd love to build a true GitOps Terraform system for you! 
```

Gowiem commented on content/updates/terraform-controller-overview.md:
```diff
@@ -0,0 +1,210 @@
+
+## Further Reading and Resources
+
+* Dive into the official [Weave GitOps Terraform Controller documentation](https://weaveworks.github.io/tf-controller/) for more profound knowledge.
```
```suggestion
* Dive into the official [Weave GitOps Terraform Controller documentation](https://weaveworks.github.io/tf-controller/) for more in-depth knowledge.
```

Gowiem commented on content/updates/terraform-controller-overview.md:
```diff
@@ -0,0 +1,210 @@
+![Runner of our ECR managed by Terraform Controller](/img/updates/terraform-controller-overview/tf_controller_3.png)
+
+
+The example of the Runner logs:
```
@gberenice this image is pretty small type in the website preview. Maybe we just include a few lines of the logs in a code block? What do you think?

gberenice replied:
Updated. Let me know if that looks better to you.

Gowiem commented on content/updates/terraform-controller-overview.md:
```diff
@@ -0,0 +1,210 @@
+
+It's important to note that, as of now, multiple runners are not automatically released in tandem with each new version of the Terraform Controller. This issue was highlighted in a [GitHub discussion](https://github.com/weaveworks/tf-controller/issues/1023) and we're looking forward to fixing this.
+
+We'd also love to mention the potential support for [OpenTofu](https://opentofu.org/) in the Terraform Controller. The GitHub issue [#1132 on the Weaveworks Terraform Controller repository](https://github.com/weaveworks/tf-controller/issues/1132) responds to the growing interest in OpenTofu following HashiCorp's decision to change Terraform's licensing. OpenTofu, a fork of Terraform, remains open ...
```
```suggestion
We also love to see the upcoming potential support for [OpenTofu](https://opentofu.org/) in the Terraform Controller. The GitHub issue [#1132 on the Weaveworks Terraform Controller repository](https://github.com/weaveworks/tf-controller/issues/1132) responds to the growing interest in OpenTofu following HashiCorp's decision to change Terraform's licensing. OpenTofu, a fork of Terraform, remains open-source and is managed by the Linux Foundation, making it an appealing alternative for the community.
```

Gowiem commented on content/updates/terraform-controller-overview.md:
```diff
@@ -0,0 +1,210 @@
+image: /img/updates/terraform-controller-overview/tf_controller_0.png
+callout: "<p>👋 <b>Interested in platform engineering for your organization</b>, but not sure where to start? <a href='/contact'>Get in touch,</a> we're an expert team of platform engineers who deliver high-quality cloud platforms for startups and SMBs looking to scale. We enable your application engineers to focus on your product and in turn generate more value for your business.</p><a href='/con ...
+---
+<h2>Table of Contents</h2>
```
Doesn't seem like this TOC is rendering correctly in the preview... we may need to cut it unless you know how it should be working?

gberenice replied:
I was missing an empty row there, this is fixed. However, these extra spaces in the least important nested headings drive me nuts. I've noticed the similar behavior in the text, so I guess it's something we can fix on netlify side (?).
<img width="678" alt="Screenshot 2024-01-30 at 16 04 20" src="https://github.com/masterpointio/masterpoint.io/assets/30597968/6a50f764-5143-4564-978f-a0a933b6b7d1">

I love TOCs, they really help to evaluate if I'm interested in the content. But I have doubts if we should release it like that.

Gowiem reviewed (changes requested):
@gberenice few nit picks on wording and what not, but overall this is fantastic. Check it out and let me know when you're ready for the final review!

gberenice commented:
> @gberenice few nit picks on wording and what not, but overall this is fantastic. Check it out and let me know when you're ready for the final review!

Thanks a lot! Good calls 💯  This definitely helps me to "fell" the language in the text better.

I've added an example of `tfctl get` that I've found in my folder with the post materials. Let me know how that looks.

Gowiem commented on content/updates/terraform-controller-overview.md:
```suggestion
The Terraform Controller could be a real game-changer in managing infrastructure as code via GitOps. We believe with some further investment and improvements that it would be solid competitor to the primary open-source Terraform automation tool, [Atlantis](https://www.runatlantis.io/). Still, we acknowledge that building a strong system around this tool requires effort and our suggestion is to only entertain using this tool if your org is ready for some level of bleeding-edge investment themselves. If you're intrigued by that idea, we welcome any engineering orgs who are eager to operationalize this tool internally to [reach out and chat with us](https://masterpoint.io/contact/) as we'd love to help you build a true GitOps Terraform system!
```

Gowiem commented on content/updates/terraform-controller-overview.md:
```suggestion
Overall, we're still on a mission to explore the evolving landscape of GitOps and Terraform, and we need your help! [Get in touch with us on LinkedIn](https://www.linkedin.com/company/masterpoint-consulting/) so you can share your experiences + thoughts so we can work together to make them even better!
```

Gowiem commented on content/updates/terraform-controller-overview.md:
```suggestion
* Dive into the official [Weaveworks GitOps Terraform Controller documentation](https://weaveworks.github.io/tf-controller/) for more in-depth knowledge.
```

Gowiem reviewed (changes requested):
@gberenice just did another pass -- Looks great and I think with these changes this is ready to ship! If it sounds good to you, we'll push it live on Thursday next week once your changes are in.

gberenice commented on content/updates/terraform-controller-overview.md:
Doh!

gberenice commented:
> @gberenice just did another pass -- Looks great and I think with these changes this is ready to ship! If it sounds good to you, we'll push it live on Thursday next week once your changes are in.

Sounds good - the date is added 🚀 
I've added your suggestions, fixed some minor typos + consolidated Weave -> Weaveworks naming everywhere.

Gowiem reviewed (approved):
Approved for merge on 02/08!
